frontend/src/components/common/LineChart.vue

```diff
@@ -22,6 +32,7 @@
 
         data() {
             return {
+                chartDate: this.data,
```

Comment:
chartDate: this.data  // It seems to me that the error is hidden here, because the data is not received (((

Reply:
get rid of chartDate